src/Data/Food/Builder/Recipe.elm

```diff
@@ -212,26 +269,27 @@ encodeTransform p =
         ]
 
 
-fromQuery : Db -> Query -> Result String Recipe
-fromQuery db query =
+fromQuery : Db -> List Country -> Query -> Result String Recipe
+fromQuery db countries query =
```

Comment:
As previously suggested, countries should probably live in the Db. Tip: always think of `Db` as your unique source for db data dependencies.